docs/platforms/apple/common/profiling/index.mdx

```diff
@@ -90,3 +90,19 @@ To set up launch profiling, use the `enableAppLaunchProfiling` option and config
 If you use `SentryOptions.tracesSampler` or `SentryOptions.profilesSampler`, it will be invoked after you call `SentrySDK.startWithOptions`, with `SentryTransactionContext.forNextAppLaunch` set to `true` indicating that it's evaluating a launch profile sampling decision. If instead you simply set `SentryOptions.tracesSampleRate` and `SentryOptions.profilesSampleRate`, those numerical rates will be used directly.
 
 Currently, launch profiles are attached to a special performance transaction operation called `app.launch` and displayed in the product simply as `launch`.
+
+## Continuous Profiling
+
+<Note>
+
+This feature is experimental and may have bugs.
+
+</Note>
+
+The current profiling implementation stops the profiler automatically after 30 seconds, if you do not stop it earlier. Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit to how long the profiler may run.
+
+Whereas profiles previously only ran in tandem with a performance transaction that you'd either start manually with `SentrySDK.startTransaction` or with an automatic transaction, now you can start and stop the profiler directly with `SentrySDK.startProfiler` and `SentrySDK.stopProfiler`. Launch profiling is another way to start it, by simply setting `SentryOptions.enableAppLaunchProfiling = true` in your call to `SentrySDK.startWithOptions`.
+
+These new APIs do not offer any sampling functionality–every call to start the profiler will start it, and the same goes for launch profiles if you've configured that. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.
```

Comment:
```suggestion
These new APIs do not offer any sampling functionality—every call to start the profiler will start it, and the same goes for launch profiles if you've configured that. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.
```

Reply:
Thank you–I need to figure out how to get Emacs to insert em dashes. It's right there in the name!